src/components/shared/Button/ActionButton.tsx

```diff
@@ -55,15 +55,13 @@ const ActionButton = ({
         typeof values == 'function' ? await values() : values;
       result = await asyncFunction(asyncFuncValues);
       if (isMountedRef.current) setLoading(false);
+      if (typeof onSuccess == 'function') onSuccess(result);
     } catch (err) {
       setLoading(false);
 
       /**
        * @todo : display error somewhere
        */
-      return;
-    } finally {
-      if (typeof onSuccess == 'function') onSuccess(result);
```

Comment:
This is a stupid bug I introduced when porting decisions, incorrectly assuming that the catch return would skip the finally block.